### DISCUSSION TITLE
CA Referral Codes & Referral counts

### DISCUSSION THREAD

**Discussion by Durgesh4993** [Girlscript-WOB/Contributor#6] [Announcements]
## Referral counts Till 20 Sep 2024

| Name                                 | Institution                                                   | Code     |
|--------------------------------------|--------------------------------------------------------------|----------|
| Yashwanth S                          | Vidyavardhaka College of Engineering                         | GWOB001  |
| Gokil P                              | Kg College of Art's and Science                              | GWOB002  |
| M S Abhishek                         | Shobhit Institute of Engineering and Technology              | GWOB003  |
| Ramakrushna Biswal                   | GIET University, Gunupur                                     | GWOB004  |
| Ahmet Abdullah                       | Nişantaşı University                                         | GWOB005  |
| Neal Gandhi                          | L.D College of Engineering                                   | GWOB006  |
| Anubha Mahapatro                    | Siksha 'O' Anusandhan                                       | GWOB007  |
| Varsha U N                           | Waiting for counselling                                      | GWOB008  |
| Anshul Nimore                        | Institute of Engineering and Technology DAVV, Indore        | GWOB009  |
| Tushti Savarn                        | Banasthali Vidyapeeth                                       | GWOB010  |
| Chetan Choudhari                     | Smt Indira Gandhi College of Engineering                    | GWOB011  |
| Sushant Soni                        | Indian Institute of Engineering Science and Technology Shibpur | GWOB012  |
| Velavan V                           | Puducherry Technological University                           | GWOB013  |
| Sameeksha Rai                        | NIT Warangal                                                | GWOB014  |
| Deepak Geetkar                       | PIEMR                                                       | GWOB015  |
| Anushka Sharma                       | International Institute of Professional Studies              | GWOB016  |
| Pranshu Sharma                       | Devi Ahilya Vishwavidyalaya                                  | GWOB017  |
| Utkarsh Arjariya                     | Samrat Ashok Technological Institute                         | GWOB018  |
| Pranjal Birla                        | SVVV                                                       | GWOB019  |
| Kalpana Patidar                      | International Institute of Professional Studies              | GWOB020  |
| Jayaprakash Biswal                   | Trident Academy of Technology                                 | GWOB021  |
| Siva Samhitha Vasipalli             | Sree Vidyanikethan Engineering College                      | GWOB022  |
| Pooja Patel                          | Sage University                                             | GWOB023  |
| Adrita Chakraborty                   | RCC Institute of Technology                                   | GWOB024  |
| Mansi Ruhil                          | Banasthali Vidyapeeth, Rajasthan                             | GWOB025  |
| Mradul Natani                        | IIPS, DAVV                                                  | GWOB026  |
| Garima Jha                           | Banasthali Vidyapith                                       | GWOB027  |
| Vanshika Raina                       | The National Institute of Engineering                        | GWOB028  |
| Nishu Pathak                         | Chaudhary Devi Lal University                                | GWOB029  |
| Naveen Vashist                       | IIPS(DAVV)                                                 | GWOB030  |
| Vikas Patel                          | Sage University Indore                                       | GWOB031  |
| Nikhitha Alla                        | CMR College of Engineering and Technology                    | GWOB032  |
| Annanya Singh                        | Chandigarh Group of Colleges                                 | GWOB033  |
| Diya Choudhary                       | Chandigarh Group Of Colleges, Landran, Punjab              | GWOB034  |
| Trisha                               | Chandigarh Group of College                                  | GWOB035  |
| Rashmi Sharma                        | Chandigarh Group of Colleges                                 | GWOB036  |
| Naina Rana                           | Chandigarh Group of Colleges                                 | GWOB037  |
| Shreya Sharma                        | Chandigarh Group of Colleges Landran                         | GWOB038  |
| Rakshitha V                          | Sri Siddhartha Institute of Technology Tumkur               | GWOB039  |
| Bhuwan                               | National Institute of Technology, Durgapur                   | GWOB040  |
| Vaibhavee Chilhate                  | Indore Institute of Science And Technology                   | GWOB041  |
| Avanish Garg                         | Acropolis Institute of Technology and Research               | GWOB042  |
| Nupur Agarwal                        | Manipal University Jaipur                                     | GWOB043  |
| Abhishek Agarwal                     | Bennett University                                           | GWOB044  |
| Ishita Agrawal                       | IIPS-DAVV                                                   | GWOB045  |
| Vanshi R Gatti                       | Nitte Institute of Professional Education                     | GWOB046  |
| Ishani Nag                           | SRM University, Sonepat                                     | GWOB047  |
| Arun R                               | University of Visvesvaraya College of Engineering (UVCE)    | GWOB048  |
| Om Jasoliya                          | Parul University                                            | GWOB049  |
| Saniya Shaikh                        | SRM University                                              | GWOB050  |
| Aayushi Goel                         | Shobhit Institute of Engineering & Technology                | GWOB051  |
| Harsh Kanojiya                      | Lakshmi Narain College of Technology and Science             | GWOB052  |
| Rebecca Adebayo                      | University of Abuja                                         | GWOB053  |
| Shayani Banerjee                     | Sister Nivedita University                                   | GWOB054  |
| Ankit Bhandari                       | Graphic Era University                                       | GWOB055  |
| Ankit Modanwal                       | IEC College of Engineering and Technology                    | GWOB056  |
| Muhammad Hasnain                     | University of Sindh                                         | GWOB057  |
| Saara                                | ADGIPS                                                     | GWOB058  |
| Priya Ghosal                         | Netaji Subhash Engineering College                           | GWOB059  |
| Balkrishna Dixit                     | Institute of Technology and Management Gwalior              | GWOB060  |
| Prayas Gondane                       | Vishwakarma Institute of Technology, Pune                   | GWOB061  |
| Vedika Dixit                         | LNCT                                                       | GWOB062  |
| Laxmi Prajapati                      | Saraswati Vidya Mandir                                      | GWOB063  |
| Subhoshri Pal                        | IIEST Shibpur                                              | GWOB064  |
| Mohammad Sahil                       | Kalinga Institute of Industrial Technology                   | GWOB065  |
| Yash Singhal                         | KIET Group of Institutions                                   | GWOB066  |
| Patel Nayanbhai                     | Government Engineering College Rajkot                        | GWOB067  |
| Saneha Rani                          | Banasthali Vidyapeeth                                       | GWOB068  |
| Abhinay Singh                        | Laxmi Institute of Technology                                | GWOB069  |
| Kashish Kushwah                      | Institute of Technology and Management                       | GWOB070  |
| Nandini Patel                        | International Institute of Professional Studies, DAVV       | GWOB071  |
| Kartikey Pandey                      | Rajkiya Engineering College, Pratapgarh Uttar-Pradesh      | GWOB072  |
| Mohit Prajapati                      | Institute of Technology and Management                       | GWOB073  |
| Meghaa K                             | Sea College of Engineering and Technology                    | GWOB074  |
| Madhuri Gupta                        | International Institute of Professional Studies, DAVV Indore | GWOB075  |
| Abhirup Datta Khan                  | Ramakrishna Mission Vivekananda Centenary College          | GWOB076  |
| Sakshi Sharma                        | Institute of Technology and Management                       | GWOB077  |
| Virengiri Goswami                    | GEC Rajkot (GTU)                                          | GWOB078  |
| Aditya Singh                         | Bharati Vidyapeeth College of Engineering Pune              | GWOB079  |
| Anshita Soni                        | IIPS DAVV                                                   | GWOB080  |
| Maria Schoinaki                      | Athens University of Economics and Business                  | GWOB081  |
| Ishika Jindal                        | Sri Guru Tegh Bahadur Institute of Management and Information Technology | GWOB082  |
| Subham Mohanty                       | IIT Tirupati                                                | GWOB083  |
| Vardhman Gupta                       | GL Bajaj Institute of Technology and Management              | GWOB084  |
| Hari Varshney                        | Dr. Akhilesh Das Gupta Institute Of Professional Studies    | GWOB085  |
| Kunal Saukhiya                      | Swami Keshvanand Institute of Technology                     | GWOB086  |
| Abhishek Agrawal                     | Bennett University                                           | GWOB087  |
| Mohammed Munazir Ansari              | East Point College of Engineering and Technology            | GWOB088  |
| Ashish Ujjwal                        | Ambalika Institute of Management and Technology              | GWOB089  |
| Vijay Yadav                          | KIET Group of Institutions                                   | GWOB090  |
| Sheshank Singh                       | Vidyalankar Institute of Technology                           | GWOB091  |
| Prashant Kumar                       | Gautam Buddha University                                      | GWOB092  |
| Ajay Gahlot                         | Gautam Buddha University                                      | GWOB093  |
| Garima Singh                         | JS Academy of Technical Education                             | GWOB094  |
| Kaniska Pal                          | Vellore Institute of Technology, Andhra Pradesh             | GWOB095  |
| Chaheti Jha                         | RV College of Engineering                                    | GWOB096  |
| Abhisek Panda                        | GIET University Gunupur                                      | GWOB097  |
| Kunal Maurya                         | AIMT Lucknow                                                | GWOB098  |
| Priyadarshi Chandrakant              | G H Raisoni College of Engineering and Technology           | GWOB099  |
| Eshita Agarwal                       | Indian Institute of Information Technology                   | GWOB100  |
| Ramineni Lakshmi naga Chaitanya      | KKR and KSR Institute of Technology and Sciences             | GWOB101  |
| Pooja                                | ADGIPS                                                       | GWOB102  |
| Prajwal D P                          | University of Visvesvaraya College of Engineering            | GWOB103  |
| Soumen Bhunia                        | Brainware University                                          | GWOB104  |
| Sayantan Patra                      | Siksha 'O' Anusandhan University                             | GWOB105  |
| Ivan Wang                            | University at Buffalo                                        | GWOB106  |
| Aditi Nagar                          | Amity University Noida                                       | GWOB107  |
| Agnibha Chakraborty                  | VIT Bhopal University                                        | GWOB108  |
| Shreya Ghodake                       | Sinhgad College of Engineering, Vadgaon BK, Pune            | GWOB109  |
| Tanveer Alam                         | Goel Institute of Technology and Management                 | GWOB110  |
| Soumya Manjunath Naik                | Srinivas Institute of Technology                             | GWOB111  |
| Tushar Kumar Chamlikar              | Jai Narain College of Technology                             | GWOB112  |
| Nagarajan V                          | Karpagam College of Engineering                              | GWOB113  |
| Aryan Kumar                          | Faculty of Engineering and Technology, University of Lucknow | GWOB114  |
| Samruddhi Dargode                   | MKSSS's Cummins College of Engineering for Women, Pune      | GWOB115  |
| Korra Vasantha                       | Vardhaman College of Engineering                             | GWOB116  |
| Kanishk Chaudhary                   | SPPU University                                             | GWOB117  |
| Mohammed Ahmed Majid                 | Aligarh Muslim University                                    | GWOB118  |
| Ojas Arora                           | University Institute of Engineering and Technology, Punjab University, Chandigarh | GWOB119  |
| Kartikey Pandey                      | Rajkiya Engineering College                                  | GWOB120  |
| Harsh Saini                          | Quantum University                                           | GWOB121  |
| Lakshmi P                            | JSS Science and Technology University                        | GWOB122  |
| Ujjwal Saini                         | Guru Tegh Bahadur Institute of Technology                    | GWOB123  |
| Chinta Sai Madhurya                 | Velagapudi Ramakrishna Siddhartha Engineering College       | GWOB124  |
| Prem Kolte                           | VIT Bhopal University                                        | GWOB125  |
| Niharika Behera                      | IIIT Bhubaneswar                                            | GWOB126  |
| Hari Varshney                        | Dr. Akhilesh Das Gupta Institute of Professional Studies    | GWOB127  |
| Jothika Manaswini Poluru            | Velagapudi Ramakrishna Siddhartha Engineering College       | GWOB128  |
| Siraj Uddin Qureshi                 | Beijing University of Technology, Beijing, China            | GWOB129  |
| Duggirala Mouli                     | Velagapudi Ramakrishna Siddhartha Engineering College       | GWOB130  |
| Siya P Kurandwad                     | R.V. College of Engineering                                  | GWOB131  |
| Harshit Sharma                       | GNIOT                                                       | GWOB132  |
| Vanshika                             | United College of Engineering and Research, Prayagraj      | GWOB133  |
| Sudhanshu Patil                     | Birla Institute of Technology and Sciences - Pilani, Hyderabad Campus | GWOB134  |
| Rishav Kumar Sinha                  | National Institute of Technology, Silchar                   | GWOB135  |
| Krish Agrawal                       | Birla Institute of Technology, Mesra                        | GWOB136  |
| Rakshitha D V                        | PES College of Engineering, Mandya                          | GWOB137  |
| Aikik Patra                         | Guru Nanak Institute of Technology                           | GWOB138  |
| Harman Deep Singh                   | Vikrant Institute of Technology and Management              | GWOB139  |
| Shivi Shukla                        | University Institute of Technology - RGPV, Bhopal          | GWOB140  |
| Gouri Kumari                        | Bikaner Technical University                                 | GWOB141  |
| Deekshith B Gowda                   | Cambridge Institute of Technology, North Campus             | GWOB142  |
| Gungun Jain                         | National Institute of Technology Karnataka, Surathkal       | GWOB143  |
| Farha Kousar                        | KNS Institute of Technology                                  | GWOB144  |
| Shivam Mishra                       | Kashi Institute of Technology                                | GWOB145  |
| Menna Zaied                         | Banha Information and Technology University                  | GWOB146  |
| Jaya Prakash Sangem                 | BV Raju Institute of Technology, Narsapur                  | GWOB147  |
| Manmohan Singh                      | CGC College of Engineering, Landran                         | GWOB148  |
| Swagatika Parida                    | Chhatrapati Shahuji Maharaj University                      | GWOB149  |
| Aditya Dashrath Dhaygude            | Padmabhoshan Vasantdada Patil Institute of Technology      | GWOB150  |
| Shweta Tiwari                       | Kashi Institute of Technology                                | GWOB151  |
| Manya Pandey                        | Jabalpur Engineering College                                 | GWOB152  |
| Harish Bonde                        | JSPM's Rajarshi Shahu College of Engineering               | GWOB153  |
| Jyoti Kumari                        | Gopal Narayan Singh University                               | GWOB154  |
| Moinak Niyogi                       | Chandigarh University                                        | GWOB155  |
| Vishal Maurya                       | Prasad Institute of Technology                                | GWOB156  |
| Ankit Barnawal                      | Gautam Buddha University                                     | GWOB157  |
| Shreyash Srivastava                | VIT Bhopal University                                        | GWOB158  |
| Akriti Singh                        | IIEST Shibpur                                               | GWOB159  |
| Vansh Chaurasiya                    | Lovely Professional University                                | GWOB160  |
| Shubhi Soni                         | Rajasthan Technical University                               | GWOB161  |
| Disha Bisht                         | Banasthali Vidyapeeth                                       | GWOB162  |
| Cheryl Lobo                         | Fr. C Rodrigues Institute of Technology, Vashi              | GWOB163  |
| Sagufta Afrin                       | Galgotias College of Engineering and Technology             | GWOB164  |
| Kunal Maurya                        | AIMT Lucknow                                               | GWOB165  |
| KLN SriValli                        | Vignan's Institute of Information Technology                 | GWOB166  |
| Macha Siri Chandana                 | BVRIT Hyderabad College of Engineering for Women           | GWOB167  |

**Comment by Durgesh4993** [comment]
## Wrong Refferals: 

- Name :  Kartik gurjar
- Referral Codes: GWOB124
- Reason: Non web 3.0 related Prroject

## 
- Name :  Shubham 
- Referral Codes: GWOB016
- Reason: Non web 3.0 related Prroject

## 
- Name: Saransh Prajapati
- Referral Code: GWOB1508
- Reason: Non allotted referral code

## 
- Name: Nandini Modi
- Referral Code: GWOBO99
- Reason: Non allotted referral code

**Comment by Durgesh4993** [comment]
Check sr no. 76

**Comment by Coder-Kartikey** [comment]
Refferal code: GWOB072 not present in the pdf

**Reply by Durgesh4993** [reply]
Check Sr. No: 72

**Reply by Durgesh4993** [reply]
Top to 15

**Comment by Durgesh4993** [comment]
# Mentors
Here are the referral codes with their counts and participant details:

| Referral Code | Count | Name                                   | Institution                                             |
|---------------|-------|----------------------------------------|--------------------------------------------------------|
| GWOB045       | 2     | Ishita Agrawal                        | IIPS-DAVV                                             |
| GWOB004       | 1     | Ramakrushna Biswal                    | GIET University, Gunupur                             |
| GWOB006       | 1     | Neal Gandhi                           | L.D College of Engineering                             |
| GWOB008       | 4     | Varsha U N                            | Waiting for counselling                                 |
| GWOB010       | 1     | Tushti Savarn                         | Banasthali Vidyapeeth                                  |
| GWOB014       | 2     | Sameeksha Rai                         | NIT Warangal                                          |
| GWOB016       | 1     | Anushka Sharma                        | International Institute of Professional Studies        |
| GWOB043       | 1     | Nupur Agarwal                         | Manipal University Jaipur                              |
| GWOB072       | 1     | Kartikey Pandey                       | Rajkiya Engineering College, Pratapgarh Uttar-Pradesh |
| GWOB099       | 1     | Kunal Maurya                          | AIMT Lucknow                                          |
| GWOB117       | 2     | Kanishk Chaudhary                    | SPPU University                                        |
| GWOB124       | 3     | Chinta Sai Madhurya                   | Velagapudi Ramakrishna Siddhartha Engineering College  |
| GWOB133       | 1     | Vanshika                              | United College of Engineering and Research Prayagraj  |
| GWOB149       | 1     | Swagatika Parida                     | Chhatrapati Shahuji Maharaj University                |
| GWOB107       | 2     | Aditi Nagar                           | Amity University Noida                                |
| GWOB126       | 1     | Niharika Behera                       | IIIT Bhubaneswar                                      |

## Instructions

- The table shows the referral codes, count, names, and respective institutions.
- To contribute or update, feel free to fork the repository and submit a pull request.

**Comment by Durgesh4993** [comment]
# Project Admins

Here are the updated referral codes with their counts and participant details:

| Referral Code | Count | Name                                   | Institution                                             |
|---------------|-------|----------------------------------------|--------------------------------------------------------|
| GWOB045       | 1     | Ishita Agrawal                        | IIPS-DAVV                                              |
| GWOB004       | 1     | Ramakrushna Biswal                    | GIET University, Gunupur                               |
| GWOB006       | 1     | Neal Gandhi                           | L.D College of Engineering                             |
| GWOB008       | 4     | Varsha U N                            | Waiting for counselling                                |
| GWOB010       | 1     | Tushti Savarn                         | Banasthali Vidyapeeth                                  |
| GWOB014       | 1     | Sameeksha Rai                         | NIT Warangal                                           |
| GWOB016       | 3     | Anushka Sharma                        | International Institute of Professional Studies        |
| GWOB043       | 1     | Nupur Agarwal                         | Manipal University Jaipur                              |
| GWOB047       | 1     | To be filled                          | Unknown                                                |
| GWOB052       | 1     | To be filled                          | Unknown                                                |
| GWOB072       | 1     | Kartikey Pandey                       | Rajkiya Engineering College, Pratapgarh Uttar-Pradesh  |
| GWOB099       | 1     | Kunal Maurya                          | AIMT Lucknow                                           |
| GWOB117       | 2     | Kanishk Chaudhary                     | SPPU University                                        |
| GWOB124       | 1     | Chinta Sai Madhurya                   | Velagapudi Ramakrishna Siddhartha Engineering College   |
| GWOB133       | 1     | Vanshika                              | United College of Engineering and Research Prayagraj   |
| GWOB149       | 1     | Swagatika Parida                      | Chhatrapati Shahuji Maharaj University                 |
| GWOB107       | 2     | Aditi Nagar                           | Amity University Noida                                 |
| GWOB126       | 1     | Niharika Behera                       | IIIT Bhubaneswar                                       |
| GWOB025       | 1     | To be filled                          | Unknown                                                |

## Instructions

- The table shows the referral codes, count, names, and respective institutions.
- To contribute or update, feel free to fork the repository and submit a pull request.

**Comment by Durgesh4993** [comment]
# Contributors

This updated version consolidates and tracks the referral codes, their counts, and participant information.

| Referral Code | Count | Name                                   | Institution                                             |
|---------------|-------|----------------------------------------|--------------------------------------------------------|
| GWOB001       | 9     | -                                      | -                                                      |
| GWOB008       | 39    | Varsha U N                             | Waiting for counselling                                |
| GWOB008       | 3     | Varsha U N                             | -                                                      |
| GWOB016       | 7     | Anushka Sharma                         | International Institute of Professional Studies        |
| GWOB022       | 4     | -                                      | -                                                      |
| GWOB025       | 1     | To be filled                           | -                                                      |
| GWOB029       | 3     | -                                      | -                                                      |
| GWOB040       | 1     | -                                      | -                                                      |
| GWOB045       | 1     | Ishita Agrawal                         | IIPS-DAVV                                              |
| GWOB048       | 5     | -                                      | -                                                      |
| GWOB048       | 2     | -                                      | -                                                      |
| GWOB055       | 7     | -                                      | -                                                      |
| GWOB055       | 3     | -                                      | -                                                      |
| GWOB059       | 9     | -                                      | -                                                      |
| GWOB059       | 6     | -                                      | -                                                      |
| GWOB064       | 2     | -                                      | -                                                      |
| GWOB076       | 4     | -                                      | -                                                      |
| GWOB076       | 2     | -                                      | -                                                      |
| GWOB081       | 7     | -                                      | -                                                      |
| GWOB107       | 4     | Aditi Nagar                            | Amity University Noida                                 |
| GWOB107       | 1     | Aditi Nagar                            | -                                                      |
| GWOB117       | 5     | Kanishk Chaudhary                      | SPPU University                                        |
| GWOB117       | 1     | Kanishk Chaudhary                      | -                                                      |
| GWOB123       | 1     | -                                      | -                                                      |
| GWOB124       | 4     | Chinta Sai Madhurya                    | Velagapudi Ramakrishna Siddhartha Engineering College   |
| GWOB126       | 6     | Niharika Behera                        | IIIT Bhubaneswar                                       |
| GWOB126       | 4     | Niharika Behera                        | -                                                      |
| GWOB126       | 2     | Niharika Behera                        | -                                                      |
| GWOB138       | 2     | -                                      | -                                                      |
| GWOB138       | 1     | -                                      | -                                                      |
| GWOB139       | 7     | -                                      | -                                                      |
| GWOB139       | 1     | -                                      | -                                                      |
| GWOB158       | 2     | -                                      | -                                                      |
| GWOB160       | 11    | -                                      | -                                                      |
| GWOB160       | 2     | -                                      | -                                                      |
| GWOB163       | 1     | -                                      | -                                                      |
| GWOB2024VA    | 36    | -                                      | -                                                      |
| GWOBMV009     | 3     | -                                      | -                                                      |
| GWOBO99       | 3     | -                                      | -                                                      |

## Instructions

- The referral codes and corresponding counts are listed above. 
- Updates or missing details can be filled in via contributions.
- Fork the repository and submit a pull request for updates.
